src/MulanPSL-1.0.xml

```diff
@@ -0,0 +1,156 @@
+   <license isOsiApproved="false" licenseId="MulanPSL-1.0" name="Mulan Permissive Software License, Version 1" listVersionAdded="3.7">
+      <crossRefs>
+         <crossRef>https://license.coscl.org.cn/MulanPSL/</crossRef>
+         <crossRef>https://github.com/yuwenlong/longphp/blob/25dfb70cc2a466dc4bb55ba30901cbce08d164b5/LICENSE</crossRef>
+      </crossRefs>
+      <notes>Both the Chinese and English translations of this license have been included in the markup, because current examples of use appear to include both translations in license documents.</notes>
+    <text>
+        <titleText><p>木兰宽松许可证, 第1版</p></titleText>
+
+        <titleText><p>木兰宽松许可证， 第1版</p>
+
+        <p>2019年8月 http://license.coscl.org.cn/MulanPSL</p></titleText>
+
+        <p>您对“软件”的复制、使用、修改及分发受木兰宽松许可证，第1版（“本许可证”）的如下条款的约束：</p>
+
+        <list>
+            <item>
+                <bullet>0.</bullet> 定义
+                <p>“软件”是指由“贡献”构成的许可在“本许可证”下的程序和相关文档的集合。</p>
+                <p>“贡献者”是指将受版权法保护的作品许可在“本许可证”下的自然人或“法人实体”。</p>
+                <p>“法人实体”是指提交贡献的机构及其“关联实体”。</p>
+                <p>“关联实体”是指，对“本许可证”下的一方而言，控制、受控制或与其共同受控制的机构，此处的控制是指有受控方或共同受控方至少50%直接或间接的投票权、资金或其他有价证券。</p>
+                <p>“贡献”是指由任一“贡献者”许可在“本许可证”下的受版权法保护的作品。</p>
+            </item>
+
+            <item>
+                <bullet>1.</bullet> 授予版权许可
+                <p>每个“贡献者”根据“本许可证”授予您永久性的、全球性的、免费的、非独占的、不可撤销的版权许可，您可以复制、使用、修改、分发其“贡献”，不论修改与否。</p>
+            </item>
+
+            <item>
+                <bullet>2.</bullet> 授予专利许可
+
+                <p>每个“贡献者”根据“本许可证”授予您永久性的、全球性的、免费的、非独占的、不可撤销的（根据本条规定撤销除外）专利许可，供您制造、委托制造、使用、许诺销售、销售、进口其“贡献”或以其他方式转移其“贡献”。前述专利许可仅限于“贡献者”现在或将来拥有或控制的其“贡献”本身或其“贡献”与许可“贡献”时的“软件”结合而将必然会侵犯的专利权利要求，不包括仅因您或他人修改“贡献”或其他结合而将必然会侵犯到的专利权利要求。如您或您的“关联实体”直接或间接地（包括通过代理、专利被许可人或受让人），就“软件”或其中的“贡献”对任何人发起专利侵权诉讼（包括反诉或交叉诉讼）或其他专利维权行动，指控其侵犯专利权，则“本许可证”授予您对“软件”的专利许可自您提起诉讼或发起维权行动之日终止。</p>
+            </item>
+
+            <item>
+                <bullet>3.</bullet> 无商标许可
+
+                <p>“本许可证”不提供对“贡献者”的商品名称、商标、服务标志或产品名称的商标许可，但您为满足第4条规定的声明义务而必须使用除外。</p>
+            </item>
+
+            <item>
+                <bullet>4.</bullet> 分发限制
+
+                <p>您可以在任何媒介中将“软件”以源程序形式或可执行形式重新分发，不论修改与否，但您必须向接收者提供“本许可证”的副本，并保留“软件”中的版权、商标、专利及免责声明。</p>
+            </item>
+
+            <item>
+                <bullet>5.</bullet> 免责声明与责任限制
+
+                <p>“软件”及其中的“贡献”在提供时不带任何明示或默示的担保。在任何情况下，“贡献者”或版权所有者不对任何人因使用“软件”或其中的“贡献”而引发的任何直接或间接损失承担责任，不论因何种原因导致或者基于何种法律理论,即使其曾被建议有此种损失的可能性。</p>
+            </item>
+        </list>
+
+        <p>条款结束</p>
+
+        <p>如何将木兰宽松许可证，第1版，应用到您的软件</p>
+
+        <p>如果您希望将木兰宽松许可证，第1版，应用到您的新软件，为了方便接收者查阅，建议您完成如下三步：</p>
+
+        <list>
+            <item><bullet>1，</bullet> 请您补充如下声明中的空白，包括软件名、软件的首次发表年份以及您作为版权人的名字；</item>
+
+            <item><bullet>2，</bullet> 请您在软件包的一级目录下创建以“LICENSE”为名的文件，将整个许可证文本放入该文件中；</item>
+
+            <item><bullet>3，</bullet> 请将如下声明文本放入每个源文件的头部注释中。</item>
+        </list>
+
+        <p>Copyright (c) [2019] [name of copyright holder]</p>
```

Comment:
I think this part is the standard header??

Reply:
I... think so? Partly I wasn't sure whether or where to flag it because it appears twice in the document -- once here and once down below the English translation. Perhaps the standard header is something we could add in later in a future release if this license gets more widespread use  :)

Reply:
I've added the standardLicenseHeader tag -- will wait to confirm the CI tests pass.